src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacExtensionDefaultAttribute/DacExtensionDefaultAttributeAnalyzer.cs

```diff
@@ -51,16 +51,17 @@ private static Task AnalyzePropertyAsync(SymbolAnalysisContext symbolContext, PX
 
 			symbolContext.CancellationToken.ThrowIfCancellationRequested();
 
-			bool isBoundField = attributeInformation.ContainsBoundAttributes(attributes.Select(a => a.AttributeClass));
+			BoundType isBoundField = attributeInformation.ContainsBoundAttributes(attributes.Select(a => a));
```

Comment:
What is the purpose of an empty `Select` here? `Select(a => a)` doesn't do any transformations.